src/user/controller/by-href.ts

```diff
@@ -13,6 +13,7 @@ class UserByHrefController extends Controller {
 
     let hasControl = false;
     let hasPassword = false;
+    let hasPrivilege = false;
```

Comment:
`hasPrivilege` is not a good description because there are many types of privileges. `isAdmin` might be more appropriate here.